Documents/Changelog/Changelog.md

```diff
@@ -3,6 +3,7 @@
 =======
 
 ## 2024-11-xx - Build 2411 - November 2024
+* Resolved [#1388](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1502) Fixes some problems creating workspaces introduced through warnings removal.
```

Comment:
Some comment, 
i.e. the Issue number in the link needs to be modified as well

Reply:
Good catch (copy paste....)

Reply:
@Wagnerp 
And corrected.

Reply:
@giduac Sorry for the confusion, I am away at the moment and relying on the GitHub app, but the usability of it is a bit weak

Reply:
NP, should be OK now.
@Smurf-IV can merge/take care, if he's happy with it...